src/02-common-principles.md

```diff
@@ -161,6 +161,27 @@ A summary of all entities in BIDS and the order in which they MUST be
 specified is available in the [entity table](./99-appendices/04-entity-table.md)
 in the appendix.
 
+### Entity-linked file collections
+
+A set of files that are related to each other based on a repetitive acquisition of
+sequential data by changing acquisition parameters one (or multiple) at a time and/or by
+being inherent components of the same data.
+The suffix serves as an identifier for these
+collections, holding a logical grouping of constituent files.
+Within each collection, files MUST be distinguished from each other by at least one 
+entity (for example, `echo`) that corresponds to an altered acquisition parameter
+(`EchoTime`) or that defines a component relationship (for example, `part`).
+Note that these entities 
+MUST be described by the specification and the parameter changes they declare SHOULD NOT invalidate
+the definition of the accompanying suffix. For example, the use of the `echo` entity along with the 
+`T1w` suffix casts doubt on the validity of the identified contrast weighting.  
```

Comment:
Re-spacing for readability. Changing SHOULD NOT (not an RFC 2119 phrase) to MUST NOT, which I believe is the intent.

```suggestion
Note that these entities MUST be described by the specification and
the parameter changes they declare MUST NOT invalidate the definition of the accompanying suffix.
For example, the use of the `echo` entity along with the `T1w` suffix casts doubt on
the validity of the identified contrast weighting.  
```